data-raw/scripts/explore-datasets.R

```diff
@@ -0,0 +1,360 @@
+library(tidyverse)
```

Comment:
Are these scripts old that you just moved into a scripts folder? I am going to focus on the new markdowns for now but can come back and review these later.

Reply:
yes, they are older scripts that I moved into the scripts folder to tidy things up